WalletWasabi.Fluent/Views/Receive/ReceiveAddressesView.axaml

```diff
@@ -0,0 +1,17 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:c="using:WalletWasabi.Fluent.Controls"
+             xmlns:receive="using:WalletWasabi.Fluent.ViewModels.Receive"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+             x:DataType="receive:ReceiveAddressesViewModel"
+             x:CompileBindings="True"
+             x:Class="WalletWasabi.Fluent.Views.Receive.ReceiveAddressesView">
+  <c:ContentArea Title="{Binding Title}"
+                 EnableBack="True"
+                 EnableCancel="True">
+    <ListBox Items="{Binding Addresses}"
```

Comment:
Item template is not yet implemented?

Reply:
Yes, it will be implemented when we have the design of the list